check-version.js

```diff
@@ -3,7 +3,7 @@ var exec = require('child_process').exec;
 exec("npm -v", function(err, stdout, stderr) {
   if (parseFloat(stdout) < 4) {
     console.error("Using npm version: " + stdout);
-    console.error("You need to be using at least npm version 4. To upgrade use `npm install npm@4.6.1 -g`.\n");
+    console.error("You need to be using at least npm version 4. To upgrade use `npm install npm@5.3.0 -g`.\n");
```

Comment:
Are we still supporting npm 4 as well? Might that cause problems with the package-lock.json file?

Also, if we use "`npm install npm@latest`" we won't have to keep updating this script with every npm 5 release.